senders/calc_message_parts.go

```diff
@@ -1,16 +1,23 @@
 package senders
 
+import (
+	"math"
+)
+
+// percent for event
+const percentForEvent = 0.3
```

Comment:
It would be nice to extract this value to config. Otherwise — great!

Reply:
yep, sure